redash/query_runner/__init__.py

```diff
@@ -76,21 +72,21 @@ def strip_trailing_semicolon(stmt):
         return stmt
 
     def is_empty_statement(stmt):
-        strip_comments = sqlparse.filters.StripCommentsFilter()
-
         # copy statement object. `copy.deepcopy` fails to do this, so just re-parse it
         st = sqlparse.engine.FilterStack()
-        stmt = next(st.run(sqlparse.text_type(stmt)))
+        st.stmtprocess.append(sqlparse.filters.StripCommentsFilter())
+        stmt = next(st.run(str(stmt)), None)
```

Comment:
Since version 0.4.0 of sqlparse, strings consisting only of spaces are no longer considered as statements.

https://github.com/andialbrecht/sqlparse/commit/27ad2564f30f1021a96addafb7b7eb6a7504c28f